src/lib/components/StepperExample.svelte

```diff
@@ -5,46 +5,46 @@
     export let example = 0;
 </script>
 
-<div>
-    <p class="mb-2">Examples:</p>
+<div class="text-sm sm:text-base">
+    <span class="mb-2">Examples:</span>
```

Comment:
Nice renaming!

Reply:
This is not really a renaming. This is changing the paragraph tag to a span to avoid using the themes in Skeleton.